### ISSUE TITLE
Restricted interests

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7275]
**Preferred term label:**
Restricted interests

**Synonyms**
Fixated interests

**Definition (free text, please give PubMed ID)**
From DSM-V: "Highly  restricted,  fixated  interests  that  are  abnormal  in  intensity  or  focus  (e.g., strong  attachment  to  or  preoccupation  with  unusual  objects,  excessively  circumscribed or perseverative interests)."

**Parent term (use hpo.jax.org/app)**
Abnormal Interest (See issue #7274)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
Is restricted (just a few things are interesting) and fixated (excessive focus on certain things) actually synonyms? @bcoleman96

**Comment by bcoleman96** [comment]
I think so, but I can't find anything that contrasts these terms or uses them interchangeably. Are those definitions you're providing just based on normal English or is that coming from your clinical insight or experience? I imagine Audrey or Lisa could give us a quick answer to this.

**Comment by pnrobinson** [comment]
I do not have expertise here, but my intuition says that the abnormality is actually restricted and fixated interests, i.e., the fixation of one's attention to a small (reduced) set of hobbies or interests. Having Restricted interests and then as a synonym Fixated interests is not exactly the same thing and might be confusing

**Comment by bcoleman96** [comment]
That makes sense to me, but would we then want to make a parent term for "Restricted Interest"?

**Comment by pnrobinson** [comment]
added term
